docker/web/Dockerfile

```diff
@@ -38,4 +38,4 @@ COPY --chown=python:python . .
 # Use numeric ids so kubernetes identifies the user correctly
 USER 999:999
 
-RUN DJANGO_SETTINGS_MODULE=config.settings.production DJANGO_DOT_ENV_FILE=.env.tracing.sample python manage.py collectstatic --noinput
+RUN DJANGO_SETTINGS_MODULE=config.settings.production DJANGO_DOT_ENV_FILE=.env.sample python manage.py collectstatic --noinput
```

Comment:
I suppose that here should be .env instead .env.sample.

Reply:
No, as this is only a placeholder as it needs a file to exist, so it doesn't matter which one